tools/experiments/experiments.js

```diff
@@ -73,6 +73,11 @@ const EXPERIMENTS = [
     name: 'Viewability APIs for amp-analytics',
     spec: 'https://github.com/ampproject/amphtml/issues/1297#issuecomment-197441289',
   },
+  {
+    id: 'amp-sticky-ad',
+    name: 'AMP Sticky Ad',
+    spec: 'https://github.com/ampproject/amphtml/issues/2472',
```

Comment:
Added an empty .md file. But I feel the content in the link explain `amp-sticky-ad` better so I leave the link here.